components/MentorRegistration/MentorFormRegistration.js

```diff
@@ -110,7 +104,7 @@ export default function MentorForm() {
   const handleUploadImageChange = async (e) => {
     const file = e.target.files[0];
     const base64 = await convertBase64(file);
-    setFormData({ ...formData, mentorImg: { name: file.name, image: base64 } });
+    setFormData({ ...formData, image: base64 });
```

Comment:
actually we will uploading directly image on cloudinary

Reply:
ok